doc/content/devices/models/dragino-lht52/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Dragino LHT52"
+description: ""
+weight: 
+---
+
+{{< figure src="lht52.png" alt="Generic Node Sensor Edition" class="float plain">}}
+
+The Dragino [LHT52](https://www.dragino.com/products/temperature-humidity-sensor/item/199-lht52.html) is a temperature & humidity sensor and has a USB Type-C port to connect to external sensors such as an external temperature sensor.
+
+<!--more-->
+
+# Onboarding to {{% tts %}}
```

Comment:
If you deploy this doc locally, you can see that if you use #, this title won't be visible on the right hand menu. We want every title in the doc to be shown so please replace # here with ## and adjust all other titles accordingly (replace ## with ###, etc.)